File: src/m/solvers/issmmumpssolver.js
function issmmumpssolver(){
//ISSMMUMPSSOLVER - 
//
//   Usage:
//      options=issmmumpssolver(varargin);

	//Retrieve options provided in varargin. First convert arguments to array:
	var args = Array.prototype.slice.call(arguments);

	//Then process options
	var  options = new pairoptions(args.slice(1,args.length));

	//default issmoptions options
	var issmoptions={};
	issmoptions['toolkit']='issm';
	issmoptions['mat_type']=options.getfieldvalue('mat_type','mpisparse');
	issmoptions['vec_type']=options.getfieldvalue('vec_type','mpi');
	issmoptions['solver_type']=options.getfieldvalue('solver_type','mumps');

	return issmoptions;
}

File: src/m/solvers/issmmumpssolver.m
function issmoptions=issmmumpssolver(varargin)
%ISSMMUMPSSOLVER - return ISSM options for the MUMPS direct solver
%
%   Usage:
%      issmoptions=issmmumpssolver();

%retrieve options provided in varargin
options=pairoptions(varargin{:});
issmoptions=struct();

%default issmoptions options
issmoptions.toolkit='issm';
issmoptions.mat_type=getfieldvalue(options,'mat_type','mpisparse');
issmoptions.vec_type=getfieldvalue(options,'vec_type','mpi');
issmoptions.solver_type=getfieldvalue(options,'solver_type','mumps');
